lib/src/base-styles/_nx-tile.scss

```diff
@@ -169,8 +144,8 @@ $nx-tile-background-color: $nx-white;
 
```

Comment:
Doesn't look like this class is demonstrated anywhere.

Reply:
> Doesn't look like this class is demonstrated anywhere.

Doesn't apply anymore since we moved to the grid layout: https://github.com/sonatype/sonatype-react-shared-components/pull/34/commits/8f266140382cfa27762566dce6c8c9cce5c2b98c

Reply:
> Doesn't apply anymore since we moved to the grid layout: 8f26614

The class I'm commenting on here is still in the code. That commit appears to be unrelated.